slices/dbus-daemon.yaml

```diff
@@ -0,0 +1,30 @@
+package: dbus-daemon
+
+essential:
+  - dbus-daemon_copyright
+
+slices:
+  bins:
+    essential:
+      - dbus-bin_bins
+      - dbus-daemon_config
+      - dbus-session-bus-common_config
+      - libapparmor1_libs
+      - libaudit1_libs
+      - libc6_libs
+      - libcap-ng0_libs
+      - libdbus-1-3_libs
+      - libexpat1_libs
+      - libselinux1_libs
+      - libsystemd0_libs
+    contents:
+      /usr/bin/dbus-daemon:
+      /usr/bin/dbus-run-session:
+
```

Comment:
Should we create the `/var/lib/dbus` directory as well? The following are from `postinst`:

```sh
if [ "$1" = configure ]; then
    # This is idempotent, so it's OK to do every time. The system bus' init
    # script does this anyway, but you also have to do this before a session
    # bus will work on non-systemd systems, so we do this here for the
    # benefit of people starting a temporary session bus in a chroot.
    mkdir -p /var/lib/dbus
    dbus-uuidgen --ensure                                                                                               
fi
```